jbpm-wb-process-runtime/jbpm-wb-process-runtime-client/src/main/java/org/jbpm/workbench/pr/client/editors/instance/list/ProcessInstanceListViewImpl.java

```diff
@@ -196,33 +199,34 @@ public boolean clearCurrentSelection( CellPreviewEvent<ProcessInstanceSummary> e
 
                     @Override
                     public DefaultSelectionEventManager.SelectAction translateSelectionEvent( CellPreviewEvent<ProcessInstanceSummary> event ) {
+                        DefaultSelectionEventManager.SelectAction ret = DefaultSelectionEventManager.SelectAction.DEFAULT;
                         NativeEvent nativeEvent = event.getNativeEvent();
                         if ( BrowserEvents.CLICK.equals( nativeEvent.getType() ) ) {
                             // Ignore if the event didn't occur in the correct column.
-                            if ( extendedPagedTable.getColumnIndex( actionsColumn ) == event.getColumn() ) {
```

Comment:
This is an issue I detected in other perspectives as well: the `actionsColumn` member only stored a reference to the actions column which was created last (i.e. the actions column corresponding to the last filter tab of the table).

@cristianonicolai if you approve this change, we can apply it to other views.